css/custom.css

```diff
@@ -91,4 +91,5 @@
 
 .background-image {
   width: 300px;
+  height: 300px;
```

Comment:
This is not necessary. The requirement here is that height = width.

Reply:
But actually then my image is getting misaligned with Pranav's image since his image does not remain in square shape then. Can you please suggest what to do?